packages/editor-ui/src/plugins/codemirror/completions/utils.ts

```diff
@@ -41,6 +41,8 @@ export function longestCommonPrefix(...strings: string[]) {
 // this function will extract expression that is currently typed so autocomplete
 // suggestions can be matched based on it.
 function extractSubExpression(userInput: string): string {
+	if (userInput.includes('new Date(')) return userInput;
```

Comment:
Date completions have been broken since this function was introduced. Wrote tests and added a quick workaround here - ideally we should later simplify all this.